fixity/fixity.py

```diff
@@ -280,6 +279,25 @@ def main():
 
     session = Session()
 
+    logger = logging.getLogger(__name__)
+
+    # create console handler which logs stderr messages
+    stderr_handler = logging.StreamHandler(stream=sys.stderr)
+
+    # create formatter and add it to the handlers
+    if args.timestamps:
+        format_time = datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M:%S %Z")
```

Comment:
Why is it necessary to call `datetime.now` here?

Reply:
let me change it.